### PR TITLE
Update for DS3 CROSSBACK E-tense

### PR DESCRIPTION
**Pull request by am-technix**
Not sure for the third member of the list "ds:ds3:21:50", I don't know if it is correct ?
The information in the log : 
Your vehicles: ["{'vin': 'VR1UJZKXZLXXXXX', 'vehicle_id': 'xxxx', 'label': 'E-TENSE', 'brand': 'DS', 'abrp_name': None, 'battery_power': 46, 'fuel_capacity': 0, 'max_elec_consumption': 70, 'max_fuel_consumption': 30}"]

### CODE REVIEW

**Comment by flobz** [comment]
Can you check the third member here : https://api.iternio.com/1/tlm/get_carmodels_list?api_key=32b2162f-9599-4647-8139-66e9f9528370
Thanks for your contribution

**Comment by am-technix** [comment]
Thank you for the link, update done for the third member.
I think it is OK for merge
Thanks

File: libs/car_model.py [hunk 1]
```diff
@@ -59,6 +59,7 @@ def __init__(self, name, battery_power, abrp_name=None, reg=None,
     ElecModel("Mokka-e", 46, "opel:corsae:20:50", r"VXKUKZKX.*"),  # VXKUKZKXZM
     ElecModel("Zaphira-e", 68, "peugeot:etraveler:21:75:opel", r"VXEVZZKX.*"),  # VXEVZZKXZMZ
     ElecModel("E-C4", 46, "citroen:ec4:21:50", r"VR7BCZKX.*"),  # VR7BCZKXCM
+    ElecModel("DS3 Crossback e-tense", 46, "ds:3crossback:20:48", r"VR1UJZKXZL*"),  # VR1UJZKXZL
```
Comment (mr-sven) [inline libs/car_model.py:62]: the regex should end with `.*`

Comment (am-technix) [reply libs/car_model.py:62]: Corrected in another pull request
I failed to add my new commit to this pull request
Sorry... Hope this is OK now

**Comment by flobz** [comment]
#194

**Pull request closed without merging**